### DISCUSSION TITLE
I cant open the fabric api. Please Help

### DISCUSSION THREAD

**Discussion by Gerskov** [FabricMC/fabric-api#2203] [FAQ]
I have download fabric api but i cant open it up and yes i have the new java kit 18. please help

**Comment by warjort** [comment]
Your question makes no sense, if you want help you will have to be a lot more specific about what you are trying to do and what is not working, e.g. error messages, logs, etc.

The fabric api is a programming library people use to help develop mods for minecraft, it has no purpose by itself.
To actually use it you need at least the fabric loader and a launcher with of course some mods.
Maybe look at this: https://fabricmc.net/wiki/doku.php
